2017-09-04 11:49:13,424 - INFO - start convert GDB...
                       source File Path:C:\Users\esri\Desktop\customerupdate
                       output Path:D:\dataHandle\GISData\gdb\crm\clientdata
                       log Path:E:/python_project/CMB-Py/dataManage\workspace\logs
2017-09-04 11:49:26,733 - INFO - insertUpdate ** update GDB soucefile:C:\Users\esri\Desktop\customerupdate
2017-09-04 11:49:26,736 - WARNING - 
2017-09-04 11:49:26,740 - WARNING - Traceback (most recent call last):
  File "E:/python_project/CMB-Py/dataManage/serverHandler/InsertGDB_update_multi.py", line 48, in ConvertGDB
    shutil.copytree(gdbServerPath, gdbPath)
  File "C:\Python27\ArcGIS10.2\lib\shutil.py", line 169, in copytree
    names = os.listdir(src)
WindowsError: [Error 3] : 'D:\\dataHandle\\GISData\\arcgisserver\\clientdata.gdb/*.*'

2017-09-04 11:49:48,082 - INFO - start convert GDB...
                       source File Path:C:\Users\esri\Desktop\customerupdate
                       output Path:D:\dataHandle\GISData\gdb\crm\clientdata
                       log Path:E:/python_project/CMB-Py/dataManage\workspace\logs
2017-09-04 11:49:52,049 - INFO - insertUpdate ** update GDB soucefile:C:\Users\esri\Desktop\customerupdate
2017-09-04 11:50:51,375 - INFO - start convert GDB...
                       source File Path:C:\Users\esri\Desktop\customerupdate
                       output Path:D:\dataHandle\GISData\gdb\crm\clientdata
                       log Path:E:/python_project/CMB-Py/dataManage\workspace\logs
2017-09-04 11:50:55,723 - INFO - insertUpdate ** update GDB soucefile:C:\Users\esri\Desktop\customerupdate
2017-09-04 11:51:05,204 - INFO - start convert GDB...
                       source File Path:C:\Users\esri\Desktop\customerupdate
                       output Path:D:\dataHandle\GISData\gdb\crm\clientdata
                       log Path:E:/python_project/CMB-Py/dataManage\workspace\logs
2017-09-04 11:51:07,591 - INFO - insertUpdate ** update GDB soucefile:C:\Users\esri\Desktop\customerupdate
2017-09-04 11:51:38,947 - INFO - start convert GDB...
                       source File Path:C:\Users\esri\Desktop\customerupdate
                       output Path:D:\dataHandle\GISData\gdb\crm\clientdata
                       log Path:E:/python_project/CMB-Py/dataManage\workspace\logs
2017-09-04 11:51:43,285 - INFO - insertUpdate ** update GDB soucefile:C:\Users\esri\Desktop\customerupdate
2017-09-04 11:53:46,608 - INFO - start convert GDB...
                       source File Path:C:\Users\esri\Desktop\customerupdate
                       output Path:D:\dataHandle\GISData\gdb\crm\clientdata
                       log Path:E:/python_project/CMB-Py/dataManage\workspace\logs
2017-09-04 11:53:46,608 - INFO - insertUpdate ** update GDB soucefile:C:\Users\esri\Desktop\customerupdate
2017-09-04 12:16:31,374 - INFO - insert 37 row to "clientdata100" layer!
2017-09-04 12:16:34,565 - INFO - insert 551 row to "clientdata102" layer!
2017-09-04 12:16:50,786 - INFO - insert 4 row to "clientdata110" layer!
2017-09-04 12:17:06,315 - INFO - insert 55 row to "clientdata116" layer!
2017-09-04 12:18:34,460 - INFO - insert 113 row to "clientdata130" layer!
2017-09-04 12:19:13,917 - INFO - insert 240 row to "clientdata134" layer!
2017-09-04 12:19:15,282 - INFO - insertUpdate ** convert success!
